docs/getting-started/slurm-basics.md

```diff
@@ -62,7 +73,7 @@ The table below lists `sbatch` options for executing your Slurm jobs:
 | `#SBATCH -n, ‐‐ntasks=<*number*>` | A maximum of **number** tasks requested for a running job. The default is one task per node, but note that the `‐‐cpus-per-task` option will change this default. |
 | `#SBATCH -‐ntasks-per-node=<*ntasks*>` | Request that **ntasks** be invoked on each node. If used with the `‐‐ntasks` option, the `‐‐ntasks` option will take precedence and the `‐‐ntasks-per-node` will be treated as a maximum count of tasks per node. Meant to be used with the `‐‐nodes` option. |
 | `#SBATCH -o, ‐‐output=<*filename pattern*>` | Defines the output file name specified in the **filename pattern**. For example, "JOB.o%j", where the “%j” is replaced by the job ID, or "JOB.%A_%a.out”, “%A” is replaced by the job ID and “%a” with the array index. |
-| `#SBATCH -p, ‐‐partition=<*partition_names*>` | Request a specific partition for the resource allocation. If not specified, the default behavior is to allow the  Slurm controller to select the default partition. For example, `SBATCH ‐‐partition=highmem` requests jobs to run on highmem partition. |
+| `#SBATCH -p, ‐‐partition=<*partition_names*>` | Request a specific partition for the resource allocation. If not specified, the default behavior is to allow the  Slurm controller to select the default partition. For example, `SBATCH ‐‐partition=free` requests jobs to run on the free highmem partition. *Note:* all other partitions require specifying an `--account`. |
```

Comment:
s/highmem//

Reply:
Fixed in PR #21 and PR #23 via force push.